src/test/java/com/manager/voucher/domain/PercentDiscountVoucherTest.java

```diff
@@ -10,19 +12,34 @@ class PercentDiscountVoucherTest {
 
     @Test
     public void 확인FixedDiscountVoucher(){
-        voucher = new PercentDiscountVoucher(20);
+        voucher = new PercentDiscountVoucher(20, LocalDateTime.now().plusMonths(1));
         assertEquals(voucher.getClass().getSimpleName(),"PercentDiscountVoucher");
```

Comment:
```suggestion
        voucher = new PercentDiscountVoucher(20, LocalDateTime.now().plusMonths(1));

        assertEquals(voucher.getClass().getSimpleName(),"PercentDiscountVoucher");
```

기본적으로 공백은 문맥을 표현하기 위한 방법으로 활용합니다.

문맥이 바뀌면 공백으로 표현해주세요. 동료가 코드 읽기 훨씬 수월해집니다.